### ISSUE TITLE
Date of publication and FRBR levels

### ISSUE THREAD

**Issue by belett** [fnielsen/scholia#405]
Hi,

For serving as references in lexemes, I'm currently creating grammar in Breton : https://tools.wmflabs.org/scholia/topic/Q12107

Scholia is really very helpful to follow the work I've done so far but sadly, Scholia doesn't handle well the current schema decided on https://www.wikidata.org/wiki/Wikidata:WikiProject_Books (itself inspired by the FRBR standard) : the subject is at the work level but the date of publication is at the edition level (so on two different items).
Is it something that could be fixed?

**Comment by fnielsen** [comment]
Yes, I suppose the topic can be carry from the work level to the edition level. Regarding the date of publication: shouldn't the work has a publication date that correspond to the first edition?

BTW I often add P921 to editions.

**Comment by belett** [comment]
Yes, duplicating data on both level is a solution but it doesn't seem elegant to me (and the whole point of the FRBR system is to avoid data duplication... for a book with hundreds of editions, duplication will be a nightmare to maintain).

For work level, I never use 'date of publication' (if I want the date of the first edition, I go to the item about the first edition, especially as a work may have several first editions: first edition in French, first edition in English and so on) but sometimes I use 'inception' for date of creation.

Thanks for your SPARQL query which already give better results!

**Comment by aproberts** [comment]
Hello, 
I tried to add an inventor from the ancient world with a manuscript created circa 1206 to show up in Scholia, especially ranking by the early publications' category. But the problem is that his work is a manuscript, and I use 'inception' (P571) for the date of his manuscript. 
My question is how the inception date can be used in Scholia to connect later publication years to the earlier works before the printing era, using the inception date?

Here is an example of the manuscript by al-Jazari, the inventor I'm working on: https://www.wikidata.org/wiki/Q96972269 
And here is the Scholia page on 'automaton': https://scholia.toolforge.org/topic/Q787116
It would be greater if the query display for 'Earliest published works on the topic' can change so that it would include early manuscripts with the inception dates, like a label for 'Earliest works on the topic' for both inception and publication dates. 
I think those interested in the history of science will love this feature too. 

Thanks so much for looking into this modification.
